config/default.yml

```diff
@@ -2,6 +2,14 @@ Ezcater/RailsConfiguration:
   Description: 'Enforce the use of `Rails.configuration` instead of `Rails.application.config`.'
   Enabled: true
 
+Ezcater/RailsEnv:
+  Description: 'Enforce the use of `Rails.configuration.x.<foo>` instead of `Rails.env` checks.'
+  Enabled: true
+  Exclude:
+    - "config/**/*"
+    - "spec/**/*"
```

Comment:
Where do we anticipate this being used in `spec/**/*`? The only place I see it used that I think makes sense is in `spec/rails_helper.rb` (to avoid running tests in a production environment). Would it be possible to only exclude that file instead of all spec files?

Reply:
I had this same question bouncing around my head for a while. `spec/rails_helper.rb` is indeed the only place it showed up under `spec/`, so I debated excluding just that file. I then started thinking about why people would inspect `Rails.env` in `spec/` but not `app/`, and could only come up with fairly unrealistic, contrived situations where we'd want to block the use, so I went with `spec/**/*`.

Thinking about it more though, in the absence of a sensible permitted use under `spec/` it's probably better to target the one file. It doesn't stop someone doing `allow(Rails.env).to receive(...)`, but the checks against `app/` would prevent those tests from applying anyway.

Will amend! 👍